buildSrc/src/main/kotlin/com/jakewharton/rxbinding/project/KotlinGenTask.kt

```diff
@@ -226,20 +226,21 @@ open class KotlinGenTask : SourceTask() {
   class KMethod(n: MethodDeclaration) {
     private val name = n.name
     private val annotations: List<AnnotationExpr> = n.annotations
-    private val comment = n.comment?.toString()?.let { cleanUpDoc(it) }
     private val extendedClass = n.parameters[0].type.toString()
+    private val comment = n.comment?.content?.let { cleanUpDoc(it) }
     private val parameters = n.parameters.subList(1, n.parameters.size)
     private val returnType = n.type
     private val typeParameters = typeParams(n.typeParameters)
     private val kotlinType = resolveKotlinType(returnType, annotations)
 
     /** Cleans up the generated doc and translates some html to equivalent markdown for Kotlin docs */
     private fun cleanUpDoc(doc: String): String {
-      return doc.replace("<em>", "*")
+      return doc
```

Comment:
To clarify - this is all because JavaParser gives you the doc body _with_ the `*`s. I tried a few combinations of `trimMargin()` but ended up just going with this to keep it simple